cmd/clusterctl/client/repository/components.go

```diff
@@ -439,6 +446,32 @@ func fixCRDWebhookNamespaceReference(o unstructured.Unstructured, targetNamespac
 	return o, errors.Errorf("failed to patch %s CustomResourceDefinition", version)
 }
 
+// fixCertificate fixes the dnsNames of cert-manager Certificates.
+func fixCertificate(o unstructured.Unstructured, originalNamespace, targetNamespace string) (unstructured.Unstructured, error) {
```

Comment:
Can we add some more context on why this is needed here?

Reply:
Done